### ISSUE TITLE
LIMIT, OFFSET don't work when subquery is present

### ISSUE THREAD

**Issue by maxhawkins** [kyleconroy/sqlc#201]
```sql
CREATE TABLE foo (bar TEXT NOT NULL);

-- name: GetFoo :one
SELECT
    (SELECT 1) AS baz
FROM foo
WHERE
	bar = $1
LIMIT $2
OFFSET $3;
```

Produces
```go
type GetFooParams struct {
	Bar   string `json:"bar"`
	Bar_2 string `json:"bar_2"`
	Bar_3 string `json:"bar_3"`
}
```

**Comment by kyleconroy** [comment]
#209 is a hot fix for this issue, but it doesn't fix it completely

**Comment by kyleconroy** [comment]
This has been fixed on `main` and will be fixed in v1.10.0. The query now generates the following input parameters.

```
// Code generated by sqlc. DO NOT EDIT.
// source: query.sql

package querytest

import (
	"context"
)

const getFoo = `-- name: GetFoo :one
SELECT
    (SELECT 1) AS baz
FROM foo
WHERE
	bar = $1
LIMIT $2
OFFSET $3
`

type GetFooParams struct {
	Bar    string
	Limit  int32
	Offset int32
}

func (q *Queries) GetFoo(ctx context.Context, arg GetFooParams) (interface{}, error) {
	row := q.db.QueryRowContext(ctx, getFoo, arg.Bar, arg.Limit, arg.Offset)
	var baz interface{}
	err := row.Scan(&baz)
	return baz, err
}
```
